framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -912,11 +942,11 @@ public Void visitMethod(MethodTree node, Void p) {
             checkPurity(node);
 
             // Passing the whole method/constructor validates the return type
-            validateTypeOf(node);
+            validateTypeOf(node, true);
 
             // Validate types in throws clauses
             for (ExpressionTree thr : node.getThrows()) {
-                validateTypeOf(thr);
+                validateTypeOf(thr, false);
```

Comment:
add `THORWS` typeuselocation

Reply:
Should `throws` statement be handled differently than `Annotated_Type` in this context? I think, if `@A` can be applied on an Exception type in a throw statement, `@A` also makes sense to be applied on any `Annotated_Type`. As a result, I think `throws` statement should be treated the same as other `Annotated_Type` when we validates the target locations.

Reply:
deleted `throws` statement for now.